hijack/tests/test_app/settings.py

```diff
@@ -100,4 +101,7 @@
 
 AUTH_USER_MODEL = "test_app.CustomUser"
 
-SESSION_SERIALIZER = "django.contrib.sessions.serializers.PickleSerializer"
+if django.VERSION >= (4, 0):
+    SESSION_SERIALIZER = "django.contrib.sessions.serializers.JSONSerializer"
+else:
+    SESSION_SERIALIZER = "django.contrib.sessions.serializers.PickleSerializer"
```

Comment:
Looks like this is not needed at all. in this case it will follow default

Reply:
You're true! I need the opposite in fact:
Keep using pickle on django3.2 and use the default on all other versions!
Let me fix that!